packages/editor/src/lib/editor/Editor.ts

```diff
@@ -9165,6 +9165,9 @@ export class Editor extends EventEmitter<TLEventMap> {
 	/** @internal */
 	private _restoreToolId = 'select'
 
+	/** @internal */
+	private _autoWheelBehavior: 'pan' | 'zoom' = 'pan'
```

Comment:
Whether the auto behavior starts off as `pan` or `zoom` should perhaps also be configurable.

It would also be nice to remember the last state the user was in in localstorage, so the same one can be used initially when loading the page again. Not sure if this is something tldraw should do automatically, or if it should emit an event on changes and allow this state to be set, so it can be implemented outside of tldraw?